pavelib/bok_choy.py

```diff
@@ -19,6 +19,7 @@
 BOKCHOY_OPTS = [
     ('test_spec=', 't', 'Specific test to run'),
     ('fasttest', 'a', 'Skip some setup'),
+    ('skip_clean', 'C', 'Skip cleaning repository before running tests'),
```

Comment:
@clytwynec Is there a reason that the C needs to be capitalized? If not, then I'd recommend it be lowercase.